src/main/java/edu/kis/powp/jobs2d/drivers/SimpleDriverCountingVisitor.java

```diff
@@ -0,0 +1,44 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.drivers.adapter.LineDriverAdapter;
+import edu.kis.powp.jobs2d.drivers.monitoring.DriverMonitorDecorator;
+import edu.kis.powp.jobs2d.transformations.RotateTransformationDecorator;
+import edu.kis.powp.jobs2d.transformations.ScaleTransformationDecorator;
+
+public class SimpleDriverCountingVisitor implements DriverVisitor{
+    private int total;
+
+    @Override
+    public void visit(ComplexDriver driver){
+        total = driver.getChildrenSize();
```

Comment:
What if a child would be ComplexDriver? The result would be incorrect. This is to "Simple". You should do accept on all children and sum total.